### PR TITLE
metalsmith-layouts: Handles partials > removed map-handlebars-partials

### PR DESCRIPTION
**Pull request by woodyrew**
Metalsmith-layouts automatically registers all partials from within a
directory if a string is passed to `partials` with a directory.

_Note:_ **No version bump has been applied.**

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -143,7 +142,7 @@ function buildlocale (source, locale) {
     .use(layouts({
       engine: 'handlebars',
       pattern: '**/*.html',
-      partials: mapHandlebarsPartials(metalsmith, 'layouts', 'partials'),
+      partials: 'layouts/partials',
```
Comment (phillipj) [inline build.js:145]: Does `metalsmith-layouts` handle this in a windows compat manner or should you use `path.join()`?

**Comment by phillipj** [comment]
Nice, deleting code == great success :+1:

**Comment by fhemberger** [comment]
Nice, so this was finally fixed in `metalsmith-layouts`. :+1:

File: build.js (hunk 1 above)
Comment (fhemberger) [inline build.js:145]: Rest of the script uses `path.join()` so we should stick to that.

@woodyrew Can you add this small change, please? Then it's good to merge.

Comment (woodyrew) [inline build.js:145]: `metalsmith-layouts` handles the paths. See https://github.com/superwolff/metalsmith-layouts/blob/master/lib/helpers/read-partials.js#L22

`partials: path.join(__dirname, 'layouts', 'partials'),`
Does **not** work.

**Pull request merged**